ui/components/multichain-accounts/multichain-accounts-tree/multichain-accounts-tree.tsx

```diff
@@ -86,7 +72,6 @@ export const MultichainAccountsTree = ({
               size={ButtonLinkSize.Sm}
               color={TextColor.primaryDefault}
               fontWeight={FontWeight.Medium}
-              onClick={() => handleWalletDetailsClick(walletId)}
```

Comment:
For other reviewers: This component is probably not used by state 2, but we might need another pass to remove state 1 old things later on, once E2E are all using state 2 UI.